OrbitGl/App.cpp

```diff
@@ -635,42 +627,53 @@ void OrbitApp::SetClipboard(const std::string& text) {
 
 //-----------------------------------------------------------------------------
 outcome::result<void, std::string> OrbitApp::OnSaveSession(
-    const std::string& file_name) {
-  OUTCOME_TRY(Capture::SaveSession(file_name));
+    const std::string& filename) {
+  OUTCOME_TRY(Capture::SaveSession(filename));
   ListSessions();
   Refresh(DataViewType::SESSIONS);
   return outcome::success();
 }
 
 //-----------------------------------------------------------------------------
-outcome::result<void, std::string> OrbitApp::OnLoadSession(
-    const std::string& file_name) {
-  std::string file_path = file_name;
+outcome::result<void, std::string> OrbitApp::ReadSessionFromFile(
+  const std::string& filename, Session* session) {
+  std::string file_path = filename;
 
-  if (Path::GetDirectory(file_name).empty()) {
-    file_path = Path::JoinPath({Path::GetPresetPath(), file_name});
+  if (Path::GetDirectory(filename).empty()) {
+    file_path = Path::JoinPath({Path::GetPresetPath(), filename});
   }
 
-  std::ifstream file(file_path);
+  std::ifstream file(file_path, std::ios::binary);
   if (file.fail()) {
     ERROR("Loading session from \"%s\": %s", file_path, "file.fail()");
```

Comment:
nit: while you are here
```suggestion
    ERROR("Loading session from \"%s\": file.fail()", file_path);
```